docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Macro Behavior
+sidebar_label: Macros
+---
+
+## Summary
+
+The macro behavior allows configuring a list of other behaviors to invoke
+when the macro is pressed, including support for completing part of the binding list
+once the macro is released.
```

Comment:
Now that we've thorough documentation below, could this introduction be simplified? 

I think we already simplified the first half of this in previous revisions, but we still refer to "the binding list" in the second half and I'm not sure it's totally necessary. Something like the suggestion below is less for a new user to wrap their head around, whilst still hinting about the awesome release functionality (which is covered thoroughly later in the doc):

```suggestion
The macro behavior allows configuring a list of other behaviors to invoke
when the macro is pressed and/or released.
```

Reply:
Yeah, totally agree. Added the simplified version.